### PR TITLE
dev: logs an error when a user tries to verify with guest or member r…

### LINKED ISSUE #211: Initialization Role Safety
**Is your feature request related to a problem? Please describe.**
Users can use the same role for the different roles in the verification feature. For example, a user used the Hacker role for the guest, member, and hacker role in the verification system. This creates an error when trying to add the role.

**Describe the solution you'd like**
Would like to get an error and then a re-prompt for a new role, ideally with a description on why it failed!

**Describe alternatives you've considered**
It might be nice to add to the role prompt the feature we have with the emoji prompt where we give a map or collection of unavailable roles and if that role is given, then it re-prompts. Would be nice to have a custom error message!

**Additional context**

### PR DESCRIPTION
**Pull request by christylo**
# TL;DR
logs an error when a user tries to verify with guest or member roles

# Description
New Feature:
Changed function getVerificationTypes and added function getValidVerificationTypes (sorry I eslinted the whole file so there's no diffs for specific parts I changed oops)

## Why
Closes #211 

## User Changes
User will now be reprompted to verify with a different role when trying to use guestRole or memberRole

## Tests Done
manual testing lol

# Breaking Changes
none

### CODE REVIEW

File: commands/essentials/init-bot.js [hunk 1]
```diff
@@ -1,332 +1,343 @@
...
+            For example: hacker @hacker . Make sure you add nothing more to the message!`, channel, userId
+        });
+        let type = typeMsg.content.replace(/<(@&?|#)[a-z0-9]*>/, ''); // clean out any snowflakes
+        type = type.toLowerCase().trim();
+        let role = typeMsg.mentions.roles.first();
+
+        if (role.id === guestRole.id || role.id === memberRole.id) {
+            sendMsgToChannel(channel, userId, 'Guest and member roles cannot be used for verification. ' +
+                'Please try again.', 30);
+            await this.getValidVerificationTypes(channel, userId, guestRole, memberRole);
```
Comment (JPGarCar) [inline commands/essentials/init-bot.js:299]: The return of this call is never used? So this won't work?

Comment (JPGarCar) [reply commands/essentials/init-bot.js:299]: Good refactor tho!

**Review by JPGarCar — CHANGES_REQUESTED** [review]
Just check that one comment, I am not sure if I am on the right or wrong here.

File: commands/essentials/init-bot.js (hunk 1 above)
Comment (mwang2000) [reply commands/essentials/init-bot.js:299]: oh good point, it should probably be like `return await this.getValidVerificationTypes(channel, userId, guestRole, memberRole);`

Comment (christylo) [reply commands/essentials/init-bot.js]: added the return!

**Review by JPGarCar — DISMISSED** [review]
Looks good!

**Review by JPGarCar — APPROVED** [review]
Looks good! Good job with this, recursion can be a bitch. Also really nice the fact that you created a function for this feature!
